packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts

```diff
@@ -127,16 +133,7 @@ export class ModelViewerGLTFInstance extends GLTFInstance {
    * Creates a clone of the given material, and applies a patch to the
    * shader program.
    */
-  [$cloneAndPatchMaterial](
-      material: MeshStandardMaterial,
-      sourceUUIDToClonedMaterial: Map<string, Material>) {
-    // If we already cloned this material (determined by tracking the UUID of
-    // source materials that have been cloned), then return that previously
-    // cloned instance:
-    if (sourceUUIDToClonedMaterial.has(material.uuid)) {
-      return sourceUUIDToClonedMaterial.get(material.uuid)!;
-    }
-
+  static[$cloneAndPatchMaterial](material: MeshStandardMaterial): Material {
```

Comment:
I think this is fine, but would you mind explaining briefly what these changes are about?

Reply:
I refactored this to remove a random piece of dependent data from the method signature, which allows it to be called from anywhere. I did this to avoid code duplication when cloning material to create a variant.